### PR TITLE
Add 7.15 whats new

### PR DESCRIPTION
**Pull request by EamonnTP**
### Summary

What's new content in 7.15.

### Docs preview

https://observability-docs_1082.docs-preview.app.elstc.co/guide/en/observability/7.15/whats-new.html

### Related issue

Closes https://github.com/elastic/observability-dev/issues/1638

### CODE REVIEW

**Review by ollyhowell — APPROVED** [review]
It generally looked good but I pulled the repo down and made a bunch of changes locally and then pushed the commit - think I was enjoying myself too much! Mostly minor grammatical suggestions and a couple of bits where I tried to improve the clarity.

For next time I'll use the + comment method you showed me but as discussed we'll highlight this as ready to go.

**Comment by tbragin** [comment]
LGTM! Thank you :)

File: docs/en/observability/whats-new.asciidoc [hunk 1]
```diff
@@ -122,15 +122,15 @@ Logs have been integrated into various contextual experiences in previous releas
 * {observability-guide-all}/7.13/whats-new.html[7.13]: APM service instances allow direct links to the logs per container, pod or host instance.
 * {observability-guide-all}/7.12/whats-new.html[7.12]: The ability to add logs viewer to any {kib} dashboard. With the release of ECS logging libraries
 in 7.11, APM introduced logs as part of a distributed end-to-end trace view.
```
Comment (alex-fedotyev) [inline docs/en/observability/whats-new.asciidoc:124]: Could you please make ```With the release of ECS logging libraries
in 7.11, APM introduced logs as part of a distributed end-to-end trace view.``` a different bullet point and format accordingly?

**Review by bmorelli25 — APPROVED** [review]

File: docs/en/observability/whats-new.asciidoc (hunk 1 above)
Comment (EamonnTP) [reply docs/en/observability/whats-new.asciidoc]: Thanks. Updated it.

**Pull request merged**